connector/src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -47,22 +47,24 @@ case class SparkBigQueryOptions(
     createDisposition: Option[CreateDisposition] = None,
     optimizedEmptyProjection: Boolean = true,
     viewExpirationTimeInHours: Int = 24,
-    maxReadRowsRetries: Int = 3
+    maxReadRowsRetries: Int = 3,
+    accessToken: Option[String] = None
```

Comment:
please move the `accessToken` parameter to be after `optimizedEmptyProjection` so we can keep all the constants at the end.

Reply:
Done